src/test/java/org/mule/tools/apikit/MainAppScaffolderWithExistingConfigApiSyncTest.java

```diff
@@ -129,10 +205,26 @@ private ScaffoldingResult scaffoldApiSync(String raml, String ramlFolder, String
 
     ScaffoldingResult result = mainAppScaffolder.run(configurationBuilder.build());
     assertTrue(result.isSuccess());
-    assertEquals(1, result.getGeneratedConfigs().size());
+    assertEquals(expectedMuleConfigurationSize, result.getGeneratedConfigs().size());
     return result;
   }
 
+  @Test
+  public void testSimpleGenerateV10TwoFiles() throws Exception {
+    List<String> files = new ArrayList<>();
+    files.add("external-globals/api.xml");
+    files.add("external-globals/globals.xml");
+
+    ApiReference apiReference = ApiReference.create(Paths.get("scaffolder/simpleV10.raml").toString());
+    ParseResult parseResult = new ParserService().parse(apiReference);
+
+    List<MuleConfig> muleConfigsFromLocations = createMuleConfigsFromLocations(files);
+    ApiSpecification apiSpecification = parseResult.get();
+
+    System.out.println("a");
+    //    ScaffoldingConfiguration configuration = new ScaffoldingConfiguration();
```

Comment:
remove

Reply:
missing asserts?

Reply:
Removed whole test, didn't apply. This is covered in reScaffoldDifferentVersionsWithGlobal test from the same class.